.babelrc

```diff
@@ -9,5 +9,9 @@
       }
     ]
   ],
-  "plugins": [ "@babel/plugin-transform-runtime" ]
-}
+  "sourceMaps": true,
```

Comment:
`"sourceMaps": true` `"retainLines": true` are not important. I was trying to get my debugger to work that's why I added these two lines. didn't work though 😒